debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
@prakashcv13, it would appear that alien is a package provided as part of the default Debian/Ubuntu software repositories. In that sense, it is a supported package for each distro like gcc, make, autoconf, etc. In the interest of reducing future maintenance costs, I believe its use warrants investigation.